Build issue from command line

When using standard `msbuild` from command line, getting:

```
notepadpp-multireplace\src\AboutDialog.cpp(ROW,COL): error C2065: 'IDC_***': undeclared identifier
```

type errors.  Due to missing:

```
#include "DockingFeature/resource.h"
```

in 'src\AboutDialog.cpp'.

Also, getting:

```
src\MultiReplace.h(20,10): fatal error C1083: Cannot open include file: 'resource.h': No such file or directory
```

due to incorrect include, should be:

```
#include "DockingFeature\resource.h"
```

--- a/src/AboutDialog.cpp
+++ b/src/AboutDialog.cpp
@@ -18,7 +18,7 @@
 #include <windows.h>
 #include <windowsx.h>
 #include <commctrl.h>
-#include "resource.h"
+#include "DockingFeature/resource.h"
 #include "AboutDialog.h"
 
 LRESULT CALLBACK WebsiteLinkProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam, UINT_PTR uIdSubclass, DWORD_PTR dwRefData)

--- a/src/MultiReplace.cpp
+++ b/src/MultiReplace.cpp
@@ -27,7 +27,7 @@
 #include <fstream>
 #include <iostream>
 #include <map>
-#include <Notepad_plus_msgs.h>
+#include "Notepad_plus_msgs.h"
 #include <bitset>
 #include <string>
 #include <functional>

--- a/src/MultiReplace.h
+++ b/src/MultiReplace.h
@@ -17,7 +17,7 @@
 #define MULTI_REPLACE_H
 
 #include "DockingFeature\DockingDlgInterface.h"
-#include "resource.h"
+#include "DockingFeature\resource.h"
 #include <string>
 #include <vector>
 #include <map>

